rio_tiler_pds/cbers/aws/cbers4.py

```diff
@@ -35,8 +35,10 @@ class CBERSReader(MultiBandReader):
     tms: TileMatrixSet = attr.ib(default=WEB_MERCATOR_TMS)
 
     _scheme: str = "s3"
```

Comment:
What do you think about adding the `scheme` as a parameter too?

Reply:
Your the boss :-)
If you think that it is a good idea, or if you want everything to be customizable, I can update it like that...

I think I did have that thought a short while, but thought that if it is in AWS directory, and using different ways of retriving the metadata-json-file (more of a pure boto s3 call). It did not make that much sense to enable other schemes.

And if one were to use "ftp" or "http", the term "hostname" -> "bucket" would be misleading in that case. (But mayby the main case where scheme is s3, and bucket is the more appropriate name for that parameter.

Actually, isn't the "reuquester pays" buckets public?
I did use environment variables to insert the region name into the hostname rasterio constucts, and that was in what got it working for me.

Reply:
oh I forgot about the `get_object`, then it doesn't make sense. The Reader should only work with S3 prefix

Reply:
Now I checked the sentinel2.py, the other class S2L2AReader does not read the json file during construction of  the object.

This might complicate things, but one could of course later do something like this:
```
    if "s3" == self.scheme:
        json_body = get_object(self.bucket, f"{prefix}/tileInfo.json", request_pays=True)
    else:
        # Do stuff. Looking at rio_tiler, it might be like:
        src_path = ""
        with rasterio.Env(**self.gdal_config):
                jsonbody = self.reader(src_path, **reader_params):
    self.tileInfo = json.loads(jsonbody)
```
If we take this step, we might take another and inject the reader and have a single way of retirering the data.

Reply:
@vincentsarago : Is this pull request "good enough" so it could be merged? Or should I do anything more? 

Should I redo the update of black reformating all python code with the new version? (In the current version, some things ended up in somewhat strange places.

Reply:
sorry I'll take care of this today!